packages/p2p/src/pages/my-ads/create-ad-form.jsx

```diff
@@ -70,6 +71,13 @@ const CreateAdForm = ({ country_list }) => {
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, []);
 
+    const getOrderExpiryOption = () => {
+        if (order_expiry_options.includes(Number(order_payment_period_string))) {
+            return order_payment_period_string;
+        }
+        return `${Math.max(...order_expiry_options)}`;
```

Comment:
`return `${Math.max(...order_expiry_options)}`;`

is this fallback still needed now that we are getting the period from backend?